<launch>
    <arg name="robot_ip" default="127.0.0.1" />
    <arg name="is_test_server" default="false" />
    <node name="ur_ctrl_client_manager" type="ur_ctrl_man" pkg="ur_ctrl_client" 
          output="screen" >
        <param name="robot_ip" value="$(arg robot_ip)" />
        <param name="ctrl_loop_rate" value="125.0" unless="$(arg is_test_server)"/>
        <param name="ctrl_loop_rate" value="1.0" if="$(arg is_test_server)"/>
        <rosparam command="load" file="$(find ur_ctrl_client)/param/joint_names.yaml"
                  param="joint_names" />
    </node>
</launch>
